drydock/templates/drydock/k8s/secrets/image-pull-secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: image-pull-secret
+  namespace: {{ K8S_NAMESPACE }}
+type: kubernetes.io/dockerconfigjson
+data:
```

Comment:
use `stringData` instead of `data`( https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#secret-v1-core) and avoid doing the base64  encoding.